solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Home/HomeView.swift

```diff
@@ -26,6 +26,13 @@ final class HomeView: UIView {
         tableView.dataSource = self
         return tableView
     }()
+    
+    private lazy var loadingView: LoadingView = {
+        let loadingView = LoadingView()
+        loadingView.translatesAutoresizingMaskIntoConstraints = false
+        loadingView.isHidden = true
+        return loadingView
+    }()
```

Comment:
Considere adicionar o `setLoadingMessage` aqui dentro.
```suggestion
    private lazy var loadingView: LoadingView = {
        let loadingView = LoadingView()
        loadingView.setLoadingMessage("Carregando Restaurantes...")
        loadingView.translatesAutoresizingMaskIntoConstraints = false
        loadingView.isHidden = true
        return loadingView
    }()
```